<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="41936" started-at="2018-10-30T21:26:57Z" finished-at="2018-10-30T21:27:39Z">
    <groups>
    </groups>
    <test name="LoginTest" duration-ms="41936" started-at="2018-10-30T21:26:57Z" finished-at="2018-10-30T21:27:39Z">
      <class name="com.criscar.swtestacademy.swd.pageobject.LoginTests">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.criscar.swtestacademy.swd.pageobject.LoginTests@2f7a2457]" name="setup" is-config="true" duration-ms="12500" started-at="2018-10-30T21:26:57Z" finished-at="2018-10-30T21:27:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="invalidLoginTest_InvalidUserNameInvalidPassword()[pri:0, instance:com.criscar.swtestacademy.swd.pageobject.LoginTests@2f7a2457]" name="invalidLoginTest_InvalidUserNameInvalidPassword" duration-ms="18875" started-at="2018-10-30T21:27:10Z" finished-at="2018-10-30T21:27:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Girilen deÄŸer en az 6 karakter olmalÄ±dÄ±r.] but found [Girilen değer en az 6 karakter olmalıdır.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Girilen deÄŸer en az 6 karakter olmalÄ±dÄ±r.] but found [Girilen değer en az 6 karakter olmalıdır.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.criscar.swtestacademy.swd.pageobject.pages.LoginPage.verifyLoginPassword(LoginPage.java:51)
at com.criscar.swtestacademy.swd.pageobject.LoginTests.invalidLoginTest_InvalidUserNameInvalidPassword(LoginTests.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_InvalidUserNameInvalidPassword -->
        <test-method status="FAIL" signature="invalidLoginTest_EmptyUserNameEmptyPassword()[pri:1, instance:com.criscar.swtestacademy.swd.pageobject.LoginTests@2f7a2457]" name="invalidLoginTest_EmptyUserNameEmptyPassword" duration-ms="7753" started-at="2018-10-30T21:27:29Z" finished-at="2018-10-30T21:27:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [LÃ¼tfen e-posta adresinizi girin.] but found [Lütfen e-posta adresinizi girin.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [LÃ¼tfen e-posta adresinizi girin.] but found [Lütfen e-posta adresinizi girin.]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.criscar.swtestacademy.swd.pageobject.pages.LoginPage.verifyLoginUserName(LoginPage.java:46)
at com.criscar.swtestacademy.swd.pageobject.LoginTests.invalidLoginTest_EmptyUserNameEmptyPassword(LoginTests.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_EmptyUserNameEmptyPassword -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:com.criscar.swtestacademy.swd.pageobject.LoginTests@2f7a2457]" name="teardown" is-config="true" duration-ms="2652" started-at="2018-10-30T21:27:37Z" finished-at="2018-10-30T21:27:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- com.criscar.swtestacademy.swd.pageobject.LoginTests -->
    </test> <!-- LoginTest -->
  </suite> <!-- Suite -->
</testng-results>
